OpenWeatherApp/ViewControllers/SoftAskViewController.swift

```diff
@@ -0,0 +1,162 @@
+//
+//  SoftAskViewController.swift
+//  OpenWeatherApp
+//
+//  Created by Ivan De Martino on 2/19/19.
+//  Copyright © 2019 Ivan De Martino. All rights reserved.
+//
+
+import UIKit
+import CoreLocation
+
+class SoftAskViewController: UIViewController, CLLocationManagerDelegate {
+  // MARK: Properties
+  let locationManager = CLLocationManager()
+  private let containerView: UIView = {
+    let view = UIView()
+    view.translatesAutoresizingMaskIntoConstraints = false
+    view.backgroundColor = .white
+    return view
+  }()
+  private let topContainerView: UIView = {
+    let view = UIView()
+    view.translatesAutoresizingMaskIntoConstraints = false
+    return view
+  }()
+  private lazy var greetingLabel = createCustomLabel(with: "Hi", ofSize: 24)
```

Comment:
As labels "Hi" e ""Can you provide..." deve ser localizadas. 

Dá uma olhada nesse pergunta: [Stackoverflow](https://stackoverflow.com/questions/34782165/how-to-make-ios-application-localized-without-storyboard?rq=1)

Sugiro também criar uma extension para `String` para tornar o processo de localização mais simples... Algo do tipo:

```Swift
extension String {
   var localized: String {
      return NSLocalizedString(self, comment: nil)
   }
}
```